andela/ah-jarvis-frontend
# Ch setup sass processing 159961947

**What does this PR do?**
Setup SASS pre-processing

**How should this be manually tested**

- clone this project
`git clone https://github.com/andela/ah-jarvis-frontend.git`

- pull and checkout branch:
`ch-setup-sass-159961947`

- run the app
`yarn dev`
- change value in `./src/sass/index.scss`
- this should change automatically in the browser
Screenshots

**Any background context you want to provide?**
None

**What are the relevant pivotal tracker stories?**
[#159961947](url)Setup SASS processing on the client
